src/style.css

```
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir Bold";
```

```suggestion
  font-family: "Avenir";
```
font-family один и тот же можно оставить, и тогда в зависимости от жирности будет подключаться нужный шрифт

---

++